skimage/feature/corner.py

```diff
@@ -917,16 +918,35 @@ def corner_subpix(image, corners, window_size=11, alpha=0.99):
 
 def corner_peaks(image, min_distance=1, threshold_abs=None, threshold_rel=None,
                  exclude_border=True, indices=True, num_peaks=np.inf,
-                 footprint=None, labels=None, *, num_peaks_per_label=np.inf):
+                 footprint=None, labels=None, *, num_peaks_per_label=np.inf,
+                 p_norm=np.inf):
     """Find corners in corner measure response image.
 
     This differs from `skimage.feature.peak_local_max` in that it suppresses
     multiple connected peaks with the same accumulator value.
 
     Parameters
     ----------
+    image : ndarray
+        Input image.
+    min_distance : int, optional
+        The minimum distance seperating peaks. Use the ``p`` argument
+        to set the Minkowski p-norm defining the distance.
     * : *
         See :py:meth:`skimage.feature.peak_local_max`.
```

Comment:
Should this line be removed? I think so, it was here just because we hadn't written the Parameters section.

Reply:
Should I copy the doc from `peak_local_max`?

Reply:
oh sorry I did not understand that we still had undocumented arguments. No keep it like this then, I will mark the conversation as resolved.